change/@ni-nimble-components-b08c8f1b-bce8-4d5e-b9f0-a3fadb6c92c9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Breaking change: Table: remove TableCellView.focusedRecycleCallback() and TableRow.closeOpenActionMenus(). The table now focuses the active cell when the table is scrolled, which has the same effect (closing open menus and blurring active cell content).",
```

I would vote to call this a patch change. I don't think I'd consider `TableCellView` or `TableRow` to be public classes or either of these methods to be part of the public API. I guess technically we should have marked them `@internal` but if we don't have any known direct clients of them then I'd feel safe pretending they have been internal the whole time.

```suggestion
  "type": "patch",
  "comment": "Table: remove TableCellView.focusedRecycleCallback() and TableRow.closeOpenActionMenus(). The table now focuses the active cell when the table is scrolled, which has the same effect (closing open menus and blurring active cell content).",
```

---

Providing a `TableCellView` instance is part of the `TableColumn` / `ColumnInternals` API so it is a public API if we say making custom columns is part of the public API and not internal APIs.

---

Marking `TableRow` internal makes sense but I think we want making custom columns to be part of the public API so I wouldn't mark `TableCellView` internal. I don't feel strongly about the change being patch or major, I'd certainly find it reasonable to say our public api for implementing custom columns is "incubating" so expect breaking changes.

---

Went ahead and committed the suggested switch to patch so the PR can merge